### ISSUE TITLE
archive on Zenodo?

### ISSUE THREAD

**Issue by d70-t** [gribscan/gribscan#35]
Do we want to archive `gribscan` on Zenodo? - We'd be able to get a DOI for this project then, but do we want this?

If we do, we could do this similar to https://github.com/eurec4a/eurec4a-intake/pull/147, where the details for filling out the required forms are stored in `.zenodo.json`.

Currently, the contributors are @lkluft, @trackow and @d70-t. In case we do want to publish this on Zenodo, everyone would have to fill out their information in the `creators` object. See [docs here](https://developers.zenodo.org/#add-metadata-to-your-github-repository-release).

**Comment by trackow** [comment]
I would say yes, as I would like to cite your work somehow in the nextGEMS paper, and is easiest with a DOI :)

**Comment by lkluft** [comment]
I would argue that it is easiest with a commit id 😉 But I agree that scientists have a hard time to accept that, which might justify the creation of a DOI to prevent tedious discussions during reviews.

**Comment by trackow** [comment]
What about this semi-automatic way? 
https://docs.github.com/en/repositories/archiving-a-github-repository/referencing-and-citing-content

**Comment by d70-t** [comment]
> What about this semi-automatic way?
> https://docs.github.com/en/repositories/archiving-a-github-repository/referencing-and-citing-content

Sure, if anything than that one.

**Comment by lkluft** [comment]
We do have a `CITATION.cff` file at the top level or our repo. The Zenodo FAQs state that, if this file is present, they "make a best-effort attempt at parsing Zenodo-compatible metadata from it". So I would suggest that we update the file (@d70-t ORCiD is missing, @trackow needs to add himself) and setup a pipeline; and whatever metadata finds its way is there (authors and ORCiDs are amongst them).

**Comment by d70-t** [comment]
I've added my ORCID to the draft PR #37, and cleaned up the file a bit. Open questions left:

* [ ] do we want author's affiliations?
* [ ] are we happy with the abstract (currently "Build indices and datasets from GRIB files.")
* [ ] we should add the DOI (I'd go for the collection DOI only) to CITATION.cff, but we can only to that once we've got a first release out at Zenodo.

I've enabled this repository in Zenodo, so as soon as we've got a new Release here, it should also appear on Zenodo.
